common/views/components/PageLayout/PageLayout.tsx

```diff
@@ -253,6 +260,18 @@ const PageLayoutComponent: FunctionComponent<Props> = ({
       <div id="root">
         {apiToolbar && <ApiToolbar extraLinks={apiToolbarLinks} />}
         <CookieNotice source={url.pathname || ''} />
+
+        {skipToContentLinks.map(({ anchorId, label }) => (
+          <>
```

Comment:
Don't think we need the extra `Fragment` here?

Reply:
Ah yeah, was a leftover from testing. Let me scrub it.